src/content/release/breaking-changes/index.md

```diff
@@ -51,6 +52,7 @@ They're sorted by release and listed in alphabetical order:
 [Deprecate `ThemeData.indicatorColor` in favor of `TabBarThemeData.indicatorColor`]: /release/breaking-changes/deprecate-themedata-indicatorcolor
 [`ImageFilter.blur` default tile mode automatic selection]: /release/breaking-changes/image-filter-blur-tilemode
 [Localized messages are generated into source, not a synthetic package]: /release/breaking-changes/flutter-generate-i10n-source
+[Material Theme System Updates]: /release/breaking-changes/material-theme-system-updates
```

Comment:
```suggestion
[Material Theme System Updates - card, dialog & tab bar themes]: /release/breaking-changes/material-theme-system-updates
```

Reply:
This suggested change will break the link and therefore the build.